Webcam Pull Request

prischu opened the pull request:
*Added webcam.css
*Added webcam.mustache
*Need to link livefeed button on /Home to welcome page instead

prischu committed:
Merge pull request #1 from ipatka/master

Merge

prischu committed:
Webcam Layout

*Added webcam.css
*Added webcam.mustache
*Need to link livefeed button on /Home to welcome page instead

ipatka commented:
Let's meet tomorrow or Friday to talk about this. I'm confused about a couple things

prischu committed:
Linked Livefeed page to Homepage

*Need to figure out way to make ip address change automatically in
webcam.mustache (not hardcode webcam link)

prischu commented:
Okie dokie

prischu committed:
Link to Pi webcam page via ip variable

*Created webcam.js to get the ip variable
*Change embedded iframe webcam link to grab ip variable from webcam.js
